databricks/koalas/tests/test_series.py

```diff
@@ -742,6 +753,23 @@ def test_duplicates(self):
         self.assert_eq(pser.drop_duplicates().sort_values(),
                        kser.drop_duplicates().sort_values())
 
+    def test_where(self):
+        pser1 = pd.Series([0, 1, 2, 3, 4], name=0)
```

Comment:
Can you add a test when `compute.ops_on_diff_frames` is off? I think we can still use a scalar values for `other` such as `int`.